### ISSUE TITLE
May I ask which Windowsapp that Windows Build-In Clipboard History belongs to?

### ISSUE THREAD

**Issue by FlanneryLong** [FiniteWill/Python-Clipboard-History#1]
I want to add Windows Build-In Clipboard History into Windows server 2019, by installing Windowsapp in powershell(Microsoft Store is unavaliable in Server 2019). 
However, I don't know which Windowsapp it belongs to, Unlike Photo, ScreenSketch and so on that can be directly known from PackageName.
Hopefully you can tell me! Thank you! Though this question not much relevant to your repo...

**Comment by FiniteWill** [comment]
Unfortunately, I was also unable to find the location of it in Windows Apps as it does not have its own dedicated package name like ScreenSketch or other applications. The very few resources I found on the subject matter gave conflicting answers as to whether Clipboard History was ever available for Windows Server 2019, and, as of yet, I have not gotten an answer from Microsoft's help forums. The only answer others in tech support communities gave me were to use an alternate clipboard manager (the most common suggestion being [Ditto](ditto-cp.sourceforge.io)). Sorry that I couldn't be much help, I've made a couple of posts in various tech communities and will give an update if the answer arises.
